docker-compose.yml

```diff
@@ -64,7 +64,7 @@ services:
   #      |___/
 
   lighthouse:
-    image: sigp/lighthouse:${LIGHTHOUSE_VERSION:-v3.2.1}
+    image: sigp/lighthouse:${LIGHTHOUSE_VERSION:-v3.4.0}
```

Comment:
Note: This lighthouse release is tagged as high-priority - https://github.com/sigp/lighthouse/releases/tag/v3.4.0. They have added couple of flags to control late block reorg behavior - https://lighthouse-book.sigmaprime.io/late-block-re-orgs.html